connection/main.py

```diff
@@ -0,0 +1,33 @@
+from fastapi import FastAPI
+from logzero import logger
+from pymongo import MongoClient
```

Comment:
Perhaps showing this at init/startup for a daemon would be better that at invoke ?

Reply:
Is this approach good, or should I create new `/init` endpoint and demonstrate usage using an agent? Issue with `/init` endpoint is, when we scale daemons only one pod gets initialized and others fail to serve request